### DISCUSSION TITLE
The retail prices of the RTX 4090-48G and RTX 4080 Super-32G have been released

### DISCUSSION THREAD

**Discussion by bobqianic** [ggml-org/llama.cpp#9193] [General]
RTX 4090-48G (not 4090D): ~$3500 (USD)
RTX 4080 Super-32G: ~$2000 (USD)

Available for immediate delivery in China

**Comment by puppetm4st3r** [comment]
> RTX 4090-48G (not 4090D): ~$3500 (USD) RTX 4080 Super-32G: ~$2000 (USD)
> 
> Available for immediate delivery in China

where to buy?

**Reply by aliencaocao** [reply]
I sell now but higher price than 3500usd.

**Reply by puppetm4st3r** [reply]
> I sell now but higher price than 3500usd.

Where to buy?

**Reply by aliencaocao** [reply]
> > I sell now but higher price than 3500usd.
> 
> Where to buy?

You can email me, my email is on the github profile
